hivemind_bus_client/http.py

```
@@ -0,0 +1,321 @@
+        LOG.debug(f"registering mycroft event: {mycroft_msg_type}")
+        self.internal_bus.on(mycroft_msg_type, func)
+
+    def emit(self, message: Union[MycroftMessage, HiveMessage],
+             binary_type: HiveMindBinaryPayloadType = HiveMindBinaryPayloadType.UNDEFINED):
+        if HiveMessage.msg_type == HiveMessageType.BINARY:
+            raise NotImplementedError("binary sending not yet implemented")
+        self.send_message(message)
+
+    ###############
+    # HiveMind HTTP Api
+    def connect(self, bus=FakeBus(), protocol=None, site_id=None):
```

Ruff (0.8.2)

Do not perform function call `FakeBus` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)